src/VehicleSetup/SetupView.qml

```diff
@@ -110,6 +111,13 @@ Rectangle {
                 panelLoader.sourceComponent = messagePanelComponent
             } else {
                 panelLoader.source = vehicleComponent.setupSource
+                for(var i = 0; i < componentRepeater.count; i++) {
+                    var obj = componentRepeater.itemAt(i);
+                    if (obj.text === vehicleComponent.name) {
+                        obj.checked = true;
```

Comment:
okay.

On Wed, Apr 20, 2016 at 11:09 PM, Don Gagne <notifications@github.com>
wrote:

> In src/VehicleSetup/SetupView.qml
> <https://github.com/mavlink/qgroundcontrol/pull/3256#discussion_r60517487>
> :
>
> > @@ -110,6 +111,13 @@ Rectangle {
> >                  panelLoader.sourceComponent = messagePanelComponent
> >              } else {
> >                  panelLoader.source = vehicleComponent.setupSource
> > +                for(var i = 0; i < componentRepeater.count; i++) {
> > +                    var obj = componentRepeater.itemAt(i);
> > +                    if (obj.text === vehicleComponent.name) {
> > +                        obj.checked = true;
>
> Ah, got it thanks. I wonder if instead of doing this, the
> showVehicleComponent implementation should do it instead (checking)? Don't
> like the code being in two different far aways places. Not sure if that is
> doable or not?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mavlink/qgroundcontrol/pull/3256/files/af054687eac80d5d2da7a23928f2e36965974358#r60517487>
>